src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -102,10 +102,10 @@ public void OverflowOnFluentCall()
         {
             int numberFluentCalls = (ExecutionConditionUtil.Architecture, ExecutionConditionUtil.Configuration) switch
             {
-                (ExecutionArchitecture.x86, ExecutionConfiguration.Debug) => 510,
-                (ExecutionArchitecture.x86, ExecutionConfiguration.Release) => 1310,
-                (ExecutionArchitecture.x64, ExecutionConfiguration.Debug) => 225,
-                (ExecutionArchitecture.x64, ExecutionConfiguration.Release) => 620,
+                (ExecutionArchitecture.x86, ExecutionConfiguration.Debug) => 520, // 510
```

Comment:
>// 510 [](start = 82, length = 6)

What is the purpose of this comment? Perhaps it would be good to provide some explanation.